mir_eval/separation.py

```diff
@@ -52,9 +52,9 @@ def validate(reference_sources, estimated_sources):
 
     Parameters
     ----------
-    reference_sources : np.ndarray, shape=(nsrc, nsampl)
+    reference_sources : np.ndarray, shape=(nsrc1, nsampl1)
```

Comment:
My thinking was that we haven't yet verified this condition (which is the purpose of this function) so stating they are the same size would invalidate the purpose of verifying their sizes.